src/Tribe/Views/V2/Template/Title.php

```diff
@@ -366,7 +366,7 @@ protected function build_category_title( $title, $cat ) {
 		/**
 		 * Filters the Event Category Archive title.
 		 *
-		 * @since TBD
+		 * @since 4.9.10
```

Comment:
:no_entry_sign: There must be a single blank line after a tag group

<sub>hash: 664acb465a3f173920a31329050eac59</sub>